client/dive-common/components/Viewer.vue

```diff
@@ -839,6 +935,14 @@ export default defineComponent({
             {{ item }} {{ item === defaultCamera ? '(Default)': '' }}
           </template>
         </v-select>
+
+        <v-btn
+          color="secondary"
+          @click="tracks3d = !tracks3d"
+        >
```

Comment:
On toggle this should reset the camera area for the display.  The video/images are resized and subsequent calls to reset the camera reset it back to it's original size instead of the 50% size.

Reply:
Also since this is being used a toggle button it should probably have two states to indicate on vs off.

Reply:
I agree, that is common sense. I'll update this. Thank you

Reply:
I think it should only be visible if you have stereo config mode (being that it is stereo with your specific configuration file to enable 3D).

Reply:
Yep, let me explain better this feature.
Basically when you activate the 3D mode, it is going to query every tracks that are not filtered out, and create 3d object if it makes sense to do so, e.g, the track contains at least one detection with the `stereo3d_*` attributes with defined value. Otherwise it is going to show a gray background and nothing interesting since there's no 3D coordinates information.

Reply:
https://github.com/Kitware/dive/pull/1348/commits/9c220b89bbf30c522cd96e2f2ffd80e7d76f7440